app/scripts/lib/app-start.js

```diff
@@ -160,19 +162,14 @@ function (
     },
 
     _trackWindowOnError: function () {
-      var self = this;
-      // if startup is okay we want to log future window.onerror events
-      window.onerror = function (message /*, url, lineNumber*/) {
-        var errMsg = 'null';
-
-        if (message) {
-          errMsg = message.toString().substring(0, Constants.ONERROR_MESSAGE_LIMIT);
-        }
+      var sentryEndpoint = '//__API_KEY__@' + this._window.location.host + '/metrics-errors';
```

Comment:
yeah, to avoid making more changes to raven.js. filed an issue here: https://github.com/getsentry/raven-js/issues/346